api/engineapi/models.py

```diff
@@ -155,6 +155,82 @@ class DropperClaimant(Base):  # type: ignore
     )
 
 
+class RegisteredContract(Base):  # type: ignore
+    __tablename__ = "registered_contracts"
+    __table_args__ = (
+        UniqueConstraint(
+            "blockchain",
+            "address",
+            "contract_type",
+        ),
+    )
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid4,
+        unique=True,
+    )
+    blockchain = Column(VARCHAR(128), nullable=False, index=True)
+    address = Column(VARCHAR(256), nullable=False, index=True)
+    contract_type = Column(VARCHAR(128), nullable=False, index=True)
+    title = Column(VARCHAR(128), nullable=True)
```

Comment:
If we will set `title(nullable=False)` we will prevent posibilities when unknown contracts will be added. It also will orginize us, and we will not forget why we added one or another contract :)